docstore/drivertest/drivertest.go

```diff
@@ -155,23 +163,25 @@ func RunConformanceTests(t *testing.T, newHarness HarnessMaker, ct CodecTester,
-	t.Run("Replace", func(t *testing.T) { withCollection(t, newHarness, testReplace) })
-	t.Run("Get", func(t *testing.T) { withCollection(t, newHarness, testGet) })
-	t.Run("Delete", func(t *testing.T) { withCollection(t, newHarness, testDelete) })
-	t.Run("Update", func(t *testing.T) { withCollection(t, newHarness, testUpdate) })
-	t.Run("Data", func(t *testing.T) { withNoRevCollection(t, newHarness, testData) })
-	t.Run("MultipleActions", func(t *testing.T) { withCollection(t, newHarness, testMultipleActions) })
-	t.Run("UnorderedActions", func(t *testing.T) { withCollection(t, newHarness, testUnorderedActions) })
-	t.Run("GetQueryKeyField", func(t *testing.T) { withCollection(t, newHarness, testGetQueryKeyField) })
-	t.Run("SerializeRevision", func(t *testing.T) { withHarnessAndCollection(t, newHarness, testSerializeRevision) })
-	t.Run("ActionsOnStructWithoutRevision", func(t *testing.T) { withNoRevCollection(t, newHarness, testActionsOnStructWithoutRevision) })
-
-	t.Run("ActionsWithCompositeID", func(t *testing.T) { withTwoKeyCollection(t, newHarness, testActionsWithCompositeID) })
-	t.Run("GetQuery", func(t *testing.T) { withTwoKeyCollection(t, newHarness, testGetQuery) })
-	t.Run("DeleteQuery", func(t *testing.T) { withTwoKeyCollection(t, newHarness, testDeleteQuery) })
-	t.Run("UpdateQuery", func(t *testing.T) { withTwoKeyCollection(t, newHarness, testUpdateQuery) })
+	t.Run("Create", func(t *testing.T) { withRevCollections(t, newHarness, testCreate) })
+	t.Run("Put", func(t *testing.T) { withRevCollections(t, newHarness, testPut) })
+	t.Run("Replace", func(t *testing.T) { withRevCollections(t, newHarness, testReplace) })
+	t.Run("Get", func(t *testing.T) { withRevCollections(t, newHarness, testGet) })
+	t.Run("Delete", func(t *testing.T) { withRevCollections(t, newHarness, testDelete) })
+	t.Run("Update", func(t *testing.T) { withRevCollections(t, newHarness, testUpdate) })
+	t.Run("Data", func(t *testing.T) { withCollection(t, newHarness, SingleKey, testData) })
+	t.Run("MultipleActions", func(t *testing.T) { withRevCollections(t, newHarness, testMultipleActions) })
+	t.Run("UnorderedActions", func(t *testing.T) { withRevCollections(t, newHarness, testUnorderedActions) })
+	t.Run("GetQueryKeyField", func(t *testing.T) { withRevCollections(t, newHarness, testGetQueryKeyField) })
+	t.Run("SerializeRevision", func(t *testing.T) { withCollection(t, newHarness, SingleKey, testSerializeRevision) })
+	t.Run("ActionsOnStructNoRev", func(t *testing.T) {
+		withCollection(t, newHarness, NoRev, testActionsOnStructNoRev)
+	})
+	t.Run("ActionsWithCompositeID", func(t *testing.T) { withCollection(t, newHarness, TwoKey, testActionsWithCompositeID) })
+	t.Run("GetQuery", func(t *testing.T) { withCollection(t, newHarness, TwoKey, testGetQuery) })
+	t.Run("DeleteQuery", func(t *testing.T) { withCollection(t, newHarness, TwoKey, testDeleteQuery) })
+	t.Run("UpdateQuery", func(t *testing.T) { withCollection(t, newHarness, TwoKey, testUpdateQuery) })
```

Comment:
See above.

Reply:
I thought I replied to this, but I don't see it....just in case it got lost:

Except for the name of the revision fields, collections don't know whether they're used with revisions or not.  A Two-key collection is just one with two keys, "Game" and "Player".